internal/pkg/storageos/node.go

```diff
@@ -43,10 +43,9 @@ func (c *Client) NodeObjects() (map[string]Object, error) {
 		return e
 	}
 
-	ctx, cancel := context.WithTimeout(c.ctx, DefaultRequestTimeout)
-	defer cancel()
+	ctx = c.AddToken(ctx)
 
-	nodes, _, err := c.api.ListNodes(ctx)
+	nodes, _, err := c.api.ListNodes(c.AddToken(ctx))
```

Comment:
The token is added above. Just passing the context should be enough.

Reply:
Well spotted - fixed